src/sha256.c

```diff
@@ -133,7 +134,7 @@ void sha256_final(SHA256_CTX *ctx, BYTE hash[])
 		while (i < 64)
 			ctx->data[i++] = 0x00;
 		sha256_transform(ctx, ctx->data);
-		memset(ctx->data, 0, 56);
```

Comment:
[Question] Do you want to change other memset functions?
![image](https://user-images.githubusercontent.com/17929800/201478980-0044a6d4-c764-4636-8b3a-3500b1b19090.png)

Reply:
A quick glance is that the rest of these are just initializing state.